x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/allocation/CcrPrimaryFollowerAllocationDeciderTests.java

```diff
@@ -185,7 +185,7 @@ static Decision executeAllocation(ClusterState clusterState, ShardRouting shardR
         final AllocationDecider decider = new CcrPrimaryFollowerAllocationDecider();
         final RoutingAllocation routingAllocation = new RoutingAllocation(
             new AllocationDeciders(List.of(decider)),
-            new RoutingNodes(clusterState),
+            new RoutingNodes(clusterState.routingTable(), clusterState.nodes()),
```

Comment:
Can we use `clusterState.getRoutingNodes()` instead?

Reply:
Same answer as above, probably fine to make that change but I didn't want to change test behavior here.